connector/src/main/java/org/geysermc/connector/GeyserConnector.java

```diff
@@ -206,6 +206,9 @@ private GeyserConnector(PlatformType platformType, GeyserBootstrap bootstrap) {
             message += LanguageUtils.getLocaleStringLog("geyser.core.finish.console");
         }
         logger.info(message);
+		if(platformType == PlatformType.STANDALONE){
```

Comment:
Please replace these tab indentations with spaces. There should also be a space after `if` and before the opening bracket `{`.